### DISCUSSION TITLE
Temporarily Inactive Meetings

### DISCUSSION THREAD

**Discussion by keligirl43** [code4recovery/12-step-meeting-list#496] [Q&A] [answered]
Hi! 

We have a meeting that is showing up as temporarily inactive. We checked the spreadsheet and we can't anything in the cells that would cause this to show up inactive. Can you please help me solve this problem? Our website is https://savannahaa.com. I've attached a screenshot that shows what we are getting. 

![Patriot Group](https://user-images.githubusercontent.com/73192299/128635961-9d4898ac-412d-4c7e-ac80-6c748974f333.png)

**Comment by joshreisner** [comment, marked as answer]
Hi @keligirl43 thanks for including all the info! The issue is that the address for these meetings is `Fort Stewart, GA, USA` which is an approximate location. If you can make it a specific street address or point of interest then they'll show up as regular in person meetings like the others.

**Reply by joshreisner** [reply]
Looks like this is a military base, I'm trying to find `American Federation of Government Employees, Building #1153` on Google Maps and it's not readily apparent where that is

**Reply by joshreisner** [reply]
https://www.afgelocal1922.com mentions a union hall in building 1153 on Fort Stewart so it sounds real, just not very google-able.

**Reply by Grateful-Viv** [reply]
Would 348 West Bultman, Bldg #1157, Fort Stewart, GA 31314 work? Is that correct? I googled "fort stewart GA AA meeting" and came back with the listing from here: https://find.aageorgia.org/meetings/location/1001

**Reply by keligirl43** [reply]
Thanks so much for the responses! 

It is a military base. 

So if I use, 348 West Bultman, Bldg #1157, Fort Stewart, GA 31314, it should work?

**Reply by keligirl43** [reply]
Also, is it Fort Stewart being listed as the city that is messing it up? Fort Stewart is located in Hinesville.

**Reply by Grateful-Viv** [reply]
>So if I use, 348 West Bultman, Bldg #1157, Fort Stewart, GA 31314, it should work?

Does it map to the correct location? If so, then yes.

**Reply by kajaporter** [reply]
I do have zoom links for some of our online meetings, but if I check the "no" button for "in person," mine come up as "Temporarily Inactive."
I currently have the "in person" button checked "yes" so that it shows up as a functioning meeting. The zoom link and phone button are showing up, but the meeting listing reads "in person," which it isn't. 
Does anyone have any suggestions?
Thank you!
<img width="279" alt="Screen Shot 2021-08-17 at 11 49 42 PM" src="https://user-images.githubusercontent.com/89118432/129834089-6387fb14-1ec7-4c5f-a988-bd5f934e79d8.png">
<img width="267" alt="Screen Shot 2021-08-17 at 11 52 28 PM" src="https://user-images.githubusercontent.com/89118432/129834300-e4eb429a-77b5-44b3-bdc8-17fb387efc84.png">

**Reply by kajaporter** [reply]
Also, some of our meetings are on Zoom, but the chair doesn't want to list the link, so you have to email them for it. Is there a work-around so that the meeting isn't listed as "in person" but doesn't have to have a zoom link?

**Reply by joshreisner** [reply]
Meetings will come up as inactive if they are:

* not online (have no online info that is a recognizable Zoom / Google Meet / Skype / etc) AND 
* are not in-person (marked as Location Temporarily Closed or don't have a street address)

We don't have a "contact us for more info" kind of system, so I recommend listing those meetings on a separate page. This meeting finder is designed for publicly listed recovery meetings.

**Reply by tech2serve** [reply]
@keligirl43 I believe this answer addresses your question? And we can mark it as such to close the discussion?

@kajaporter, if you still have a concern, please open a new discussion.

Thanks!

**Comment by kajaporter** [comment]
Got it. So, is there a way to remove the “in person” designation from showing up on the listing? (see screenshot) We really want to keep the locations on the map, but we’re confusing people with listings that are temporarily on Zoom.


____________________________
Kim Porter, CFRS
Executive Director
Be a Part of the Conversation
(o) 267-629-2214 
(c) 215-416-5604
***@***.***
https://conversation.zone/

**Reply by Grateful-Viv** [reply]
You cannot have the map if you just want to list it as an online meeting. Otherwise it would consider it a hybrid or just in-person if you didn't list the online URL (or phone number).

If you are not comfortable listing the entire URL you could enter something like "https://us02web.zoom.us/j/" and indicate in the meeting note section they need to contact the group by the email listed which you would enter in the group email field. Please note that will not show up in the Meeting Guide app, so you may want to add a note they need to see the links on the actual website.

Note: I am not one of the Code for Recovery developers, but rather a trusted webservant. So for definitive information, @joshreisner or @tech2serve are the two of the trusted resources to reach out to. I was just trying to be helpful.
